bower.json

```diff
@@ -19,6 +19,7 @@
     "px-icon-set": "^2.0.0",
     "px-dropdown": "custom-trigger-slot",
     "vaadin-grid": "^4.1.0",
+    "vaadin-radio-button": "^v1.0.0-alpha8",
```

Comment:
- no need in `v` prefix
- no need in `-alpha8` suffix

"^1.0.0" will be enough